docs/docs/cmd/aad/o365group/o365group-recyclebinitem-clear.md

```diff
@@ -0,0 +1,30 @@
+# aad o365group recyclebinitem clear
+
+Clears Microsoft 365 Groups from the recycle bin in the current tenant
+
+## Usage
+
+```sh
+m365 aad o365group recyclebinitem clear [options]
+```
+
+## Options
+
+`--confirm`
+: Don't prompt for confirmation to clear the recycle bin items.
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Clears all deleted Microsoft 365 Groups in the tenant.Will prompt for confirmation before removing the group
```

Comment:
```suggestion
Clears all deleted Microsoft 365 Groups in the tenant. Will prompt for confirmation before removing the group
```

Reply:
Updated the description for both the examples.